src/linker/implementation_metadata.yaml

```diff
@@ -34,3 +34,13 @@ step_2_python_pyspark_distributed:
   path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/
   name: python_pyspark_distributed
   requires_spark: true
+step_1_r:
+  step: step_1
+  path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/
+  name: r-image
```

Comment:
Most of the images by convention do not include the "-image" suffix like the research repo does, but in this case I thought that just calling it "r" was potentially confusingly short. I don't mind changing it to r or dummy-r or something if folks have a stronger opinion.

Reply:
that's fine - it can be anything, really, and I agree that "r" is too short.